README.md

```diff
@@ -10,122 +10,108 @@ Android Auto for IDrive
 
+Supported cars
+--------------
+
+Any MY2014 or newer BMW or Mini equipped with NBT or NBT Evo and the "Navigation System Professional (S609A)" [option](https://www.mdecoder.com/) or "BMW Apps (6NR)" or "Mini Connected (SA6NM)" options should be compatible.
+
 Overview
 --------
 
 As part of the Connected Apps feature, when the phone connects over USB (or Bluetooth in 2017+ models), a tunnel is created to allow other apps on the phone to interact with the car. Over this connection, enabled phone apps can show a special interface in the car, while providing tight user integration due to actually running all logic on the phone.
 
 Android Auto for IDrive, combined with the safety benefits of the tactile IDrive controller, allows the user to interact with their incoming notifications and control their phone's music, while the phone is tucked out of reach.
 
-Getting Started
----------------
+Where can I get it ?
+--------------------
 
 This app requires that the BMW or Mini Connected app for your car is installed and can successfully add Connected and Calendar entries to your car's Connected Apps menu.
 
-Download the APK from the [Releases page](https://github.com/hufman/AndroidAutoIdrive/releases/latest). Choose the one that says "sentry" to automatically upload crash reports, or choose "nonalytics" otherwise. After starting, the app should detect the Connected app and start waiting for the car connection.
+| 🚗 Stable build | 🔧 Development build |
+|-----------------|----------------------|
+| Download the APK from the [Releases page](https://github.com/hufman/AndroidAutoIdrive/releases/latest) | [Download the latest dev build](https://androidautoidrive.s3.amazonaws.com/hufman/AndroidAutoIdrive/androidautoidrive-latest-master-nomap-nonalytics-release.apk) |
+
 
-If you'd like to try out the latest build, [download it here](https://androidautoidrive.s3.amazonaws.com/hufman/AndroidAutoIdrive/androidautoidrive-latest-master-nomap-nonalytics-release.apk)!
+For the stable build, you can download the APK that says `sentry` to automatically upload crash reports.
+Choose the `nonalytics` build if you do not want to send any crash data.
 
 Check out the [FAQ](https://hufman.github.io/AndroidAutoIdrive/faq.html) if you run into problems.
 
 User Guide
 ----------
 
-![Phone Connection](https://hufman.github.io/AndroidAutoIdrive/images/screenshot-connection.png)
+| | |
+|--|--|
+| **Connecting to the car** ||
+After connecting the phone to the car, the official Connected app should show this car icon in the status bar. When this icon appears, this app should connect and add its functionality to the car. See [this guide](https://hufman.github.io/AndroidAutoIdrive/connection.html) for tips on improving the connection reliability of the Connected app.|![Phone Connection](https://hufman.github.io/AndroidAutoIdrive/images/screenshot-connection.png)|
```

Comment:
If you have time sometime, please feel free to send me some matching pictures (just for me to perspective-correct, unless you want to do that too) from your BMW, which might be recognizable by a larger audience!

Reply:
Sure! I think I can manage to do that in the weekend, will let you know.